bot/exts/backend/branding/_repository.py

```diff
@@ -93,6 +111,18 @@ class BrandingRepository:
     def __init__(self, bot: Bot) -> None:
         self.bot = bot
 
+    def _raise_for_status(self, response: ClientResponse) -> None:
+        """
+        Raise GitHubServerError if the response status is >= 500.
+
+        The server error indicates that the request shall be retried.
+        """
+        log.trace(f"Response status: {response.status}")
+        if response.status >= 500:
+            raise GitHubServerError
```

Comment:
if the request fails 5 times with a 5xx error, currently the actual server error wont be included in the traceback, just the traceback from this line. If we do something like this would it help put more context in the traceback?
```python
try:
    response.raise_for_status()
except ResponseErrorOrWhateverItsCalled as err:
    if response.status >= 500:
        raise GitHubServerError from err
    raise
```

Reply:
That's a fantastic idea! Thanks, I'll work it in.